Lesson30/Kadir/my-app/src/components/Exercise6Component/index.jsx

```diff
@@ -1,8 +1,27 @@
-export const Exercise6Component = () => {
-const handleClick  = (event) => {
-  console.log(event);
-  event.preventDefault();
-}
-
-  return (<a href="https://www.google.com/" onClick = {handleClick}>link</a>)
-};
\ No newline at end of file
+import React from 'react';
+
+// export const Exercise6Component = () => {
+// const handleClick  = (event) => {
+//   console.log(event);
+//   event.preventDefault();
+// }
+
+//   return (<a href="https://www.google.com/" onClick = {handleClick}>link</a>)
+// };
+
+// KCS Extension
+
+export const Exercise6Component2 = () => {
+  const handleLinkClick = (event) => {
+    console.log(event.target.href);
+    event.preventDefault();
+  };
+
+  return (
+    <div>
+      <button><a href="https://www.google.com/" onClick={handleLinkClick}>
```

Comment:
It's very uncommon to wrap an <a> element inside a button element, usually you use either <a> or a <button> and you can style both of them in the same way if you want to.

```suggestion
      <a href="https://www.google.com/" onClick={handleLinkClick}>
```